libpod/container_inspect.go

```diff
@@ -698,6 +699,8 @@ func (c *Container) generateInspectContainerHostConfig(ctrSpec *spec.Spec, named
 		for cap := range boundingCaps {
 			capDrop = append(capDrop, cap)
 		}
+		// Sort CapDrop so it displays in consistent order (GH #9490)
+		sort.Strings(capDrop)
```

Comment:
Do we have to do the same for capAdd?

Reply:
CapAdd is already deterministic (it generates based on the OCI spec in the DB, which doesn't change; CapDrop generates off the list of default capabilities from the `gocapability` library, which seems to return caps in a nondeterministic order?).

Reply:
Perfect :) Thanks!